Makefile

```diff
@@ -69,3 +69,11 @@ clean:
 	@# Delete libGeNN
 	@rm -f $(LIBGENN)
 	@rm -f $(BACKEND_LIBS)
+
```

Comment:
It might be cool and convenient to have a makefile rule to setup an interactive pygenn container with  `-it`, and bind host `pwd` to some directory (e.g. 'workspace') in the working directory of the container. Just for people who don't want to use jupyter.